### PR TITLE
feat: add Common Labels Support

### PR DESCRIPTION
**Pull request by bagnaram**
Common Labels to Helm Chart
## Description

Adds optional common labels support which gets added to all deployable Kube resources.
## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [x] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

If you haven't done so yet, we would appreciate it if you could star the [OpenFGA repository](https://github.com/openfga/openfga). :)

### CODE REVIEW

**Review by jalyng — APPROVED** [review]

**Comment by hello-caleb** [comment]
Thank you for your contribution to the Helm charts! If you're comfortable with it, [we encourage contributors to update documentation](https://github.com/openfga/.github/blob/main/CONTRIBUTING.md#:~:text=The%20documentation%20should%20be%20updated%20(in%20a%20separate%2C%20linked%20PR)%2C%20but%20if%20you%27re%20not%20confident%20in%20your%20technical%20writing%20you%20may%20skip%20this%20step.) where it makes sense. In this case, you may want to mention common labels in the chart’s [README.md](https://github.com/openfga/helm-charts/blob/main/charts/openfga/README.md) and include a usage example in values.yaml to help users. If you do wish to update the documentation, you can do so in a separate PR and link it to this one.

**Comment by bagnaram** [comment]
Thank you for reviewing @hello-caleb ! I am not sure which separate documentation I need to update but I have now provided usage examples.

File: charts/openfga/values.schema.json [hunk 1]
```diff
@@ -61,6 +61,11 @@
             },
             "additionalProperties": false
         },
+        "commonLabels": {
+            "type": "object",
+            "description": "Common labels to apply to OpenFGA resources",
+            "default": ""
```
Comment (rhamzeh) [inline charts/openfga/values.schema.json:67]: The default should be `{}` here

**Review by rhamzeh — CHANGES_REQUESTED** [review]
Thanks for the PR! Added a small suggestion

**Review by rhamzeh — APPROVED** [review]

**Review by rhamzeh — APPROVED** [review]

**Pull request merged**